src/plugins/github/cliPlugin.js

```diff
@@ -87,15 +91,15 @@ export class GithubCliPlugin implements CliPlugin {
     for (const repoId of config.repoIds) {
       repositories.push(await fetchGithubRepoFromCache(repoId, {token, cache}));
     }
-    const graph = Graph.merge(
+    const wg = mergeWeightedGraph(
       repositories.map((r) => {
         const rv = new RelationalView();
         rv.addRepository(r);
         return createGraph(rv);
       })
     );
-    const weights = weightsForDeclaration(declaration);
-    return {graph, weights};
+    const decWeights = weightsForDeclaration(declaration);
+    return {graph: wg.graph, weights: mergeWeights([decWeights, wg.weights])};
```

Comment:
defaultWeights would be a more readable name

Reply:
Updated.